### ISSUE TITLE
Unable to install Yawik

### ISSUE THREAD

**Issue by carlosmtr** [cross-solution/YAWIK#392]
Hi Team

Thanks for building this awesome software.
I downloaded Yawik source code from Sourceforge link provided on release section of Yawik's github repo. I was trying to install it but "Install" button is not working. I mean nothing is happening on clicking this button. I filled all the required details:

```
Database connection string: mongodb://localhost:27017/yawikdb
Initial user name: admin
Password: admin1234
Email address for system messages: <email@example.com>
```

When i click on "Install" button, i can see a form POST with below details:

```
POST: http://127.0.0.1/?p=install&lang=en
Form data:
db_conn: mongodb://localhost:27017/yawikdb
username: admin
password: admin1234
email: <email@example.com>
```

But response is EMPTY with status code 200. Also there is no error or warning in apache's log files.

https://ibb.co/kYOqQF [Screenshot]

I am using docker due to some competiblity issues with the specific requirements of Yawik. Here is the content of my docker file:

```
FROM php:5.6-apache
RUN apt-get update && apt-get install -y apt-utils zlib1g-dev libicu-dev g++ libcurl4-openssl-dev pkg-config libssl-dev libxslt1.1 libxslt1-dev
RUN docker-php-ext-configure intl
RUN docker-php-ext-install intl
RUN pecl install mongodb
RUN docker-php-ext-enable mongodb
RUN docker-php-ext-install xsl
COPY config/yawik.com.conf /etc/apache2/sites-available/
COPY config/php.ini /usr/local/etc/php/
COPY src/ /var/www/yawik.com/
RUN a2enmod rewrite
RUN a2dissite 000-default.conf
RUN a2ensite yawik.com.conf 
```

Here is the content of yawik.com.conf:

```
<VirtualHost *:80>
    DocumentRoot /var/www/yawik.com/public
    AddDefaultCharset utf-8
    ErrorLog ${APACHE_LOG_DIR}/yawik-error.log
    CustomLog ${APACHE_LOG_DIR}/yawik-access.log combined
    SetEnv APPLICATION_ENV "development"
    <Directory /var/www/yawik.com/public>
         DirectoryIndex index.php
         Options Indexes FollowSymLinks MultiViews
         AllowOverride All
         Require all granted
    </Directory>
</VirtualHost>
```

Here is the content of php.ini referenced above:

```
upload_max_filesize = 100M
post_max_size = 108M
```

Am i missing something that is required? Looking forward for your reply.
 
PS: MongoDB is running on my localhost and listening on TCP port 27017.

**Comment by TiSiE** [comment]
Thank you for your exemplary report. 

Unfortunately I was not able to reproduce the described behavior yet. 

I used the tgz for YAWIK 0.29.2 and tried the installation process on linux with apache2 and it worked like a charm. (which is not helping you in any way, I know...)

However, an empty response is a strong indication that the php process terminated abnormally. You could try to enable the error_reporting in the php.ini, that should at least outputs something...

**Comment by pigulla** [comment]
I have the exact same problem - no output despite `error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT` and `display_errors = On`.

**Comment by TiSiE** [comment]
@pigulla 
Where did you set the configuration?

Try putting them in the php.ini used by your webserver. If it is not accessible, try setting them in the .htaccess file (if you're using apache). 

Is there anyting in the log/error.log?

**Comment by PaoloScarnati** [comment]
With Ubuntu 14 same issue (with XAMPP or Apache2 either).

**Comment by cbleek** [comment]
We've recently released 0.31.1. So let's close this issue.
